msisdn-gateway/manifest.json

```diff
@@ -16,6 +20,7 @@
 	},
 	"pre-prod": {
 	    "test_types" : [ "stack-check", "e2e-test" ],
+            "aws_region": "eu-west-1",
```

Comment:
still unsure of what our prod envs will be completely so we aren't testing them yet.  but for consistency, i could add those params in there too if you prefer.